Dropdown/Dropdown.module.less

```diff
@@ -143,35 +125,122 @@
 							border-bottom-color: @agate-dropdown-up-button-open-border-bottom-color;
 							border-top-color: @agate-dropdown-button-open-border-color;
 						}
+
+						.focus({
+							.bg {
+								border-bottom-color: @agate-dropdown-up-button-open-border-bottom-color;
+							}
+						})
 					}
 				}
+			}
+		}
 
-				.dropdownList {
-					border-radius: @agate-dropdown-up-button-list-border-radius;
-				}
+	});
+}
+
+.dropdownList {
+	.applySkins({
+		border-bottom: @agate-dropdown-border-width @agate-dropdown-border-style @agate-dropdown-border-color;
+		border-radius: @agate-dropdown-list-border-radius;
+		background-color: @agate-dropdown-list-bg-color;
+		overflow: hidden;
+
+		.item {
+			border-color: @agate-item-border-color;
+			border-width: @agate-item-border-width;
+
+			.focus({
+				border-color: @agate-item-focus-border-color;
+			});
+
+			&.selected {
+				color: @agate-item-color;
 			}
+		}
 
-			.dropdownListWithScroller {
-				.scroller {
-					// FIXME:
-					[role="button"] {
-						.focus({
-							& > div {
-								background-image: none;
-							}
-						});
-					}
+		.dropDownListItem {
+			font-size: @agate-dropdown-item-font-size;
+			font-weight: @agate-dropdown-item-font-weight;
+			line-height: @agate-dropdown-item-line-height;
 
-					[role="button"] > div {
-						border-color: transparent;
-						color: black;
-					}
+			&.radioItem {
+				border-color: @agate-dropdown-item-border-color;
+				border-width: @agate-dropdown-item-border-width;
+				padding-left: @agate-dropdown-item-padding;
+				padding-right: @agate-dropdown-item-padding;
 
-					[role="button"] + div::before {
-						background-color: black;
-					}
+				.focus({
+					background-color: @agate-dropdown-item-focus-bg-color;
+				});
+			}
+		}
+
+		&.smallest {
+			width: @agate-dropdown-list-smallest-width;
+		}
+
+		&.small {
+			width: @agate-dropdown-list-small-width;
+		}
+
+		&.medium {
+			width: @agate-dropdown-list-medium-width;
+		}
+
+		&.large {
+			width: @agate-dropdown-list-large-width;
+		}
+
+		&.x-large {
+			width: @agate-dropdown-list-x-large-width
+		}
+
+		&.huge {
+			width: @agate-dropdown-list-huge-width;
+		}
+
+		&.above {
+			border-radius: @agate-dropdown-up-button-list-border-radius;
+		}
+
+		&.dropdownListWithScroller {
+			height: @agate-dropdown-list-height;
+
+			.scroller {
+				height: 100%;
+				// FIXME:
+				[role="button"] {
+					.focus({
+						& > div {
+							background-image: none;
+						}
+					});
+				}
+
+				[role="button"] > div {
+					border-color: transparent;
+					color: black;
+				}
+
+				[role="button"] + div::before {
+					background-color: black;
 				}
 			}
 		}
 	});
 }
+
+.applySkins({
+	.contextualPopup{
+		.container {
+			padding: 0;
+
+			&.below {
+				&.small {
+					margin-top: 0;
+				}
+			}
```

Comment:
I found `offset` prop for `ContexualPopupDecorator`. I think you don't need this code and use `offset: "none"` to `DropdownButton` component.